docs/blog/v1.md

```diff
@@ -0,0 +1,188 @@
+
+Special thanks to [@jxnblk](https://jxnblk.com) and [@wooorm](https://wooorm.com) for their help improving the
+docs and updating the build tooling.
+
+## Breaking changes
+
+Unfortunately, we’ve had to introduce a few breaking changes, but we’ve written a migration guide which
+will walk you through the steps needed to upgrade.
+
+*   🚨`@mdx-js/tag` is replaced by `@mdx-js/react` and an `@mdx` pragma - [migration guide](/migrating/v1#pragma)
+*   MDXProvider now merges component contexts when nested - [migration guide](/migrating/v1#mdxprovider)
+*   React support now requires `>= 16.8` in `@mdx-js/react` - [migration guide](/migrating/v1#react)
+
+[Read the full v1 migration guide](/migrating/v1)
+
+## Deprecations
+
+Luckily, we only need to introduce one deprecation.  The `mdPlugins` and `hastPlugins` options have been renamed
+`remarkPlugins` and `rehypePlugins` respectively.  For the time being we will issue a warning when the old options
+are used.  In v2, the old options will be removed.
+
+## 📈 Growth
+
+A major release is always a good time to take a step back and reflect on what’s been happening in terms of growth
+and the greater community.  The ecosystem surrounding MDX has already grown larger than we ever dreamed.  We’ve also
+seen projects/products/application we never even imagined.
+
+### Ecosystem
+
+*   [Prettier](https://prettier.io)
+*   [Docz](https://docz.site)
+*   [MDX Deck](https://github.com/jxnblk/mdx-deck)
+*   [Next.js](https://nextjs.org)
+*   [Gatsby](https://gatsbyjs.org)
+*   [AST Explorer](https://astexplorer.net)
+*   [Vue support (alpha)](/vue)
+*   [Demoboard](https://frontarm.com/demoboard/)
+*   [Editors](/editors)
+
+### Numbers
+
+*   **Downloads**: 125,000/week, 2.4M total
+*   **Stars**: 6,200
+*   **Contributors**: 50
+*   **Pull requests**: 281
+*   **Commits**: 670
+
+The contributor growth is one of the most important aspects here as we have numerous community members that are familiar
+with MDX internals.  This allows us to continually improve the project and spread the workload against an ever growing
+team of contributors.
+
+[See the contributor graph](https://github.com/mdx-js/mdx/graphs/contributors)
+
+## 🛣 Roadmap
+
+With v1 released we have a roadmap in place that we’ll continue working towards over the next 6 months or
+so in addition to bug fixes and parsing issues we might encounter.
+
+*   MDXs: [#454](https://github.com/mdx-js/mdx/issues/454)
+*   Interleaving: [#195](https://github.com/mdx-js/mdx/issues/195)
+*   Global shortcodes: [#508](https://github.com/mdx-js/mdx/pull/508)
+*   Stable Vue support: [#238](https://github.com/mdx-js/mdx/issues/238)
+*   Blocks: MDX WYSIWYG: [blocks/blocks](https://github.com/blocks/blocks)
+*   MDX playground inspired by AST Explorer: [#220](https://github.com/mdx-js/mdx/issues/220)
+*   New splash page: [#444](https://github.com/mdx-js/mdx/issues/444)
+*   Showcase page: [#414](https://github.com/mdx-js/mdx/issues/414)
+
+### Vue support
```

Comment:
This is much cooler than being down here! It’s usable, right? So add it somewhere at the top! And if there are caveats, describe them here and link to it from the top.

Reply:
My thought on keeping it lower is that it's only an alpha and will have it's own release announcement when more thoroughly tested.